modules/push-notifications/src/admin/index.ts

```diff
@@ -97,16 +100,68 @@ export class AdminHandlers {
     return 'Ok';
   }
 
+  async getNotificationTokenByUserId(
+    call: ParsedRouterRequest,
+  ): Promise<UnparsedRouterResponse> {
+    const tokenDocuments = await NotificationToken.getInstance().findMany(
+      {
+        userId: call.request.urlParams.userId,
+      },
+      undefined,
+      undefined,
```

Comment:
what if user has more tokens than the default limit of findMany? Shouldn't we add pagination here or at least set sorting by descending creation date which I think would be the most common case.

Reply:
skip/limit defaults are not applied to rpc functions